packages/ses/src/tame-locale-methods.js

```diff
@@ -0,0 +1,54 @@
+import { assert } from './assert.js';
+import { getOwnPropertyNames, defineProperty } from './commons.js';
+
+const localePattern = /\b(\w*[a-z])[L]ocale([A-Z]\w*\b)/;
```

Comment:
The initial and final break match might be better as expressly start and end of the whole string, `/^...$/` instead of `\b...\b`, though it would be hard for a vetted shim to tease out different behavior.